features/polyExplorer/src/model/jurisdictions.js

```diff
@@ -0,0 +1,8 @@
+export const jurisdictions = {
```

Comment:
Nit: Suppose this could be a default export, given how there's only one thing being exported here, but not important.